game/theater/iadsnetwork/iadsnetwork.py

```diff
@@ -157,10 +172,10 @@ def node_for_group(self, group: IadsGroundGroup) -> IadsNetworkNode:
         self.nodes.append(node)
         return node
 
-    def node_for_tgo(self, tgo: TheaterGroundObject) -> IadsNetworkNode:
+    def node_for_tgo(self, tgo: TheaterGroundObject) -> Optional[IadsNetworkNode]:
```

Comment:
Instead of raising an exception, we consider a tgo not to be participating if None is returned